released/packages/coq-haskell/coq-haskell.1.0.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "1.0.0"
+
+homepage: "https://github.com/jwiegley/coq-haskell"
+dev-repo: "git+https://github.com/jwiegley/coq-haskell.git"
+bug-reports: "https://github.com/jwiegley/coq-haskell/issues"
+license: "BSD-3-Clause"
+
+synopsis: "A library to provide Haskell-familiar constructions in Coq"
+description: """
+This library is designed for Haskell users who are either using Coq to build
+code intended for extraction to Haskell, or who wish to prototype/prove their
+algorithms in Coq. It provides a collection of definitions and notations to
+make Gallina more familiar to Haskellers.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.10" & < "8.16~") | (= "dev")}
+]
+
+url {
+  src: "https://github.com/jwiegley/coq-haskell/archive/refs/tags/1.0.tar.gz"
+  checksum: "sha256=64b6958f4eca641b933977a047272dea00ef5ceff33206da66f10fe792dbf3f2"
+}
+
+authors: [
+  "John Wiegley"
+]
+
+tags: [
+  "keyword:haskell"
+  "date:2022-03-29"
```

Comment:
```suggestion
  "category:Computer Science/Data Types and Data Structures"
  "date:2022-03-29"
```
Maybe this could make the package easier to find?